src/interaction/interactiveTarget.js

```diff
@@ -42,5 +49,11 @@ module.exports = {
      * @todo Needs docs.
      * @private
      */
-    _touchDown: false
+    _touchDown: false,
+    
+    /**
+     * Used to track the global start position of a touchdown event on this target.
+     * @private
+     */
+    _touchDownPoint: new core.Point()
```

Comment:
@englercj the issue is that if you do not relate the object based on the touch itself then multitouch  might make this flakely, if only have slightly unexpected behavior. But I don't know to what extent without testing myself.